documentation/modules/openstack-prerequisites.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="openstack-prerequisites_{context}"]
+= OpenStack prerequisites
+
+The following prerequisites apply to {osh} migrations:
+
+* You must use a xref:compatibility-guidelines_{context}[compatible version] of {osh}.
+* You must have the {manager} CA certificate or the {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server.
++
+You can obtain the CA certificate by navigating to the `/etc/keystone/keystone.con`  file on the `keystone` server using the `tls_certfile` property.
+
+[IMPORTANT]
+====
+Migration using {osh} source providers is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[NOTE]
+====
+Migration using {osh} source providers only supports VMs whose volume snapshots are supported by {osh} Block Storage (Cinder). Migrations for VMs that use {osh} Swift at all or use only {osh} Glance for backend storage are not currently supported.
```

Comment:
@bennyz can you please check the correctness of this part?

Reply:
I think `glance for backend storage` is confusing, as glance is an image store, the idea here is convey that the VM should start from a volume rather than directly from glance, but the volume can be based on swift or glance

Reply:
@bennyz @ahadas  How about one of these? 

a) "Migration using {osp} source providers only supports VMs whose volume snapshots are supported by {osp} Block Storage (Cinder) or volumes based on {osp} Swift or {osp} Glance"  ?

b)  "Migration using {osp} source providers only supports VMs whose volumes  are supported by {osp} Block Storage (Cinder), {osp} Swift, or {osp} Glance."

Reply:
I think something like this would clear enough:
"Migration using {osp} source providers only supports VMs that uses only Cinder volumes"

Reply:
@bennyz -- Nice and clear -- thanks!